tests/api-tests/access-control/schema.test.ts

```diff
@@ -221,13 +220,6 @@ const introspectionQuery = `{
   }
 }`
 
-class FakePrismaClient {
-  $on () {}
-  async findMany () {
-    return [{ id: 'mock' }]
-  }
-}
```

Comment:
This is incompatible with how we are using Prisma now, and probably Prisma 5 too